appside/model/dbt_worksheet_model.js

```diff
@@ -56,6 +64,25 @@ class DbtWorksheetModelFwd extends Model {
         ]);
     }
 
+    /**
+     * Build likert frames for a DBT worksheet model as pre or post measurement frame.
+     */
+    build_pre_post_measurement_frames() {
```

Comment:
suggest changing name to `build_pre_post_measurement_frame` since the method only builds one frame with each call.

Reply:
fixed.